devlib/utils/ssh.py

```diff
@@ -219,6 +224,15 @@ def cancel_running_command(self):
                 return True
         return False
 
+    def connected_as_root(self):
+        pass
+
+    def set_busybox_wrapper(self, wrapper):
```

Comment:
Connections shouldn't know anything about busybox, because it's nothing to do with the actual connection